lib/util.js

```diff
@@ -62,17 +62,26 @@ exports.deprecate = function(fn, msg) {
   }
 
   var warned = false;
+  const warnedLines = new Set();
+
   function deprecated() {
+    var location = (new Error()).stack.match(/.*?\n.*\n.*?\((.*?:\d+):/)[1];
```

Comment:
@silverwind For anonymous functions, I am getting `Object.<anonymous> `. The lines without parens are for the calls directly from the modules.